Makefile

```diff
@@ -20,6 +20,20 @@ build: $(BINARY_NAME)
 $(BINARY_NAME):
 	go build -v -o $(BINARY_NAME)
 
+release:
```

Comment:
not needed, release is done via GH action. you can remove this and format the code its good to go